Sources/PackageModel/SwiftSDKs/SwiftSDKBundle.swift

```diff
@@ -150,4 +150,27 @@ extension [SwiftSDKBundle] {
     public var sortedArtifactIDs: [String] {
         self.flatMap(\.artifacts.keys).sorted()
     }
+
+    public func listSwiftSDKs(hostTriple: Triple) -> [String : [Triple]] {
```

Comment:
Is this meant to be adopted outside of SwiftPM? If not, it shouldn't be `public`. Also, please add a doc string to the function declaration.

```suggestion
    package func listSwiftSDKs(hostTriple: Triple) -> [String : [Triple]] {
```

Reply:
I feel that this is so generally useful that it should be a public API, do you disagree?

Reply:
Alright, will change it.